docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -244,6 +244,64 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+.. note::
+
+    Trino can autogenerate the ``_schema`` contents by running a ``SELECT *`` over the collection.
+    However, not all the fields are always detected and you might need to update the schema manually.
```

Comment:
Similar sentence exists in `mongodb.schema-collection` section. How about fixing there? 
Also, I would like to mention `ALTER TABLE` statement approach in addition to manual `_schema` fix.

Reply:
Moved and mentioned `ALTER TABLE`.